### PR TITLE
Added advanced security page to Properties

### LINKED ISSUE #1647: Add security tab to properties
From a feedback on Feedback Hub (Apps->Files UWP).
https://aka.ms/AA96qf4

### LINKED ISSUE #5163: Remove experimental setting to show file owner in properties dialog
With the recent addition of the security tab, this setting is no longer needed.

### LINKED ISSUE #5204: Added security options to drive properties
**Explain the bug**
Right clicking a drive and going to properties doesn't show a security tab

**Logfile**
[debug.log](https://github.com/files-community/Files/files/6644552/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/6644553/debug_fulltrust.log)

**Screenshots**
image

**System Information**
- OS Version: Windows 10 21H1 19042.1055
- App version: v1.5.32

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #1647
- Closes #5163 
- Closes #5204

**Details of Changes**
Add details of changes here.
- Implemented advanced security page for Properties
- Moved "File Owner" property to Details tab & removed setting (#5163)
- Added security tab for drives (#5204)

**Screenshots (optional)**

image

**TODO**
- [x] Yellow stuff -> this PR
- [x] Orange stuff -> next PR if needed

image

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by R3voA3** [comment]
Looks sweet! Good job.

File: Files/Filesystem/Permissions/FileSystemAccessRule.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using Newtonsoft.Json;
```
Comment (yaichenbaum) [inline Files/Filesystem/Permissions/FileSystemAccessRule.cs:1]: Is this being used?

**Comment by yaichenbaum** [comment]
@mdtauk without changing the layout too drastically (or even with changing the layout), do you have any ideas how we can polish up the UI for the advanced permission?

**Comment by mdtauk** [comment]
> @mdtauk without changing the layout too drastically (or even with changing the layout), do you have any ideas how we can polish up the UI for the advanced permission?

Nope

File: Files/Filesystem/Permissions/FileSystemAccessRule.cs (hunk 1 above)
Comment (gave92) [reply Files/Filesystem/Permissions/FileSystemAccessRule.cs]: Nope, removed

**Comment by yaichenbaum** [comment]
@gave92 Should we add support for resizing columns? It feels kind of rough right now.

**Comment by gave92** [comment]
@yair100 I'd rather not do it now, column resizing with a listview adds a lot of complexity and I don't see it as necessary here as it's for Details view. I wish we could easily reuse the Details view control and get column resizing and sorting..

**Comment by yaichenbaum** [comment]
@gave92 Can you resolve the merge conflicts?

**Review by winston-de — DISMISSED** [review]

**Review by yair100 — APPROVED** [review]

**Review by winston-de — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@gave92 Great work!

**Pull request merged**